pages/account/manage/milestone/[[...data]].js

```diff
@@ -182,75 +198,76 @@ export default function ManageMilestone({ BASE_URL, milestone }) {
                 <div className="sm:grid sm:grid-cols-3 sm:items-start sm:gap-4 sm:border-t sm:border-primary-low-medium/30 sm:pt-5">
                   <div className="mt-1 sm:col-span-2 sm:mt-0">
                     <Input
+                      {...register("title")
+                      }
                       name="title"
                       label="Milestone Title"
-                      onChange={(e) => setTitle(e.target.value)}
-                      value={title}
                       placeholder="Title of your Milestone"
-                      required
-                      minLength="2"
-                      maxLength="256"
                     />
+                    {errors.title && <p className=" text-red-500 ">{errors.title.message}</p>}
                     <p className="text-sm text-primary-low-medium">
                       For example: <i>GitHub Star</i>
                     </p>
                   </div>
 
                   <div className="mt-1 sm:col-span-2 sm:mt-0">
                     <Textarea
+                      {...register("description")
+                      }
                       name="description"
                       placeholder="Description of your Milestone"
-                      onChange={(e) => setDescription(e.target.value)}
-                      value={description}
-                      required
-                      minLength="2"
-                      maxLength="512"
                     />
+                    {errors.description && <p className=" text-red-500 ">{errors.description.message}</p>}
```

Comment:
Extra spaces not needed 
```suggestion
                    {errors.description && <p className="text-red-500">{errors.description.message}</p>}
```